web/templates/user_custom_profile_fields.hbs

```diff
@@ -1,5 +1,5 @@
 {{#each profile_fields}}
-    <li data-type="{{this.type}}" class="field-section custom_user_field" data-field-id="{{this.id}}">
+    <li data-type="{{this.type}}" class="field-section custom_user_field">
```

Comment:
Hmm, is it possible we need to be passing something here? I would kinda have expected that we would need a `data-field-id` to find which field is modified by inputs here.

Reply:
This is a great example of a question that’s difficult to answer without reading through all our code, since the global nature of CSS classes and dataset selectors make this difficult to distinguish from

https://github.com/zulip/zulip/blob/f5d8a8938e17a034ab29c7d55bbe93877a921026/web/templates/settings/custom_user_profile_field.hbs#L1

Having gone and read more code, I can see that we’re trying to reuse `settings_account.intialize_custom_user_type_fields` for both even though they’re completely different components. So I guess this should be passed.